Source/Fuse.Scripting.JavaScript/JavaScript.uno

```diff
@@ -54,6 +59,7 @@ namespace Fuse.Reactive
 
 			if(--_javaScriptCounter <= 0)
 			{
+				//TODO: probably have to block this a _moduleInstance was preserved somewhere...?
```

Comment:
uhh, what? :)

Reply:
I think there's a missing "if", "...block this if a _moduleInstance..."

I think it was my question about contexts and initialization. So far I've not seen problems in preview, but if something does some up this might be a place to look. On the short-term though, perhasp the comment can just be removed for now.

Reply:
Soo, I guess it should be removed, then?